File: ctf/src/main/java/com/efficios/jabberwocky/ctf/trace/generic/GenericCtfTraceIterator.java
package com.efficios.jabberwocky.ctf.trace.generic;

import com.efficios.jabberwocky.ctf.trace.CtfTraceIterator;
import com.efficios.jabberwocky.ctf.trace.event.CtfTraceEvent;

public class GenericCtfTraceIterator extends CtfTraceIterator<CtfTraceEvent> {

    public GenericCtfTraceIterator(GenericCtfTrace originTrace) {
        super(originTrace);
    }

}

File: jabberwocky-ctf/src/main/kotlin/com/efficios/jabberwocky/ctf/trace/generic/GenericCtfTraceIterator.kt
package com.efficios.jabberwocky.ctf.trace.generic

import com.efficios.jabberwocky.ctf.trace.CtfTraceIterator
import com.efficios.jabberwocky.ctf.trace.event.CtfTraceEvent

class GenericCtfTraceIterator(originTrace: GenericCtfTrace) : CtfTraceIterator<CtfTraceEvent>(originTrace)
